plugin/slime.vim

```diff
@@ -277,6 +282,7 @@ command -bar -nargs=0 SlimeConfig call s:SlimeConfig()
 command -range -bar -nargs=0 SlimeSend <line1>,<line2>call s:SlimeSendRange()
 command -nargs=+ SlimeSend1 call s:SlimeSend(<q-args> . "\r")
 command -nargs=+ SlimeSend0 call s:SlimeSend(<args>)
+command! SlimeSendCL call s:SlimeSendCurrentLine()
```

Comment:
How about inlining the whole line here?

    command! SlimeSendCurrentLine call s:SlimeSend(getline(".") . "\r")